.github/workflows/check-unlinked-content.js

```diff
@@ -1,133 +0,0 @@
-var fs = require("fs");
```

Comment:
This check is no longer needed, it has been implemented in our `docs-page` component as part of [react-components#220](https://github.com/hashicorp/react-components/pull/220).